src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -138,15 +120,17 @@ The Koji debugger enables live testing and debugging in a production environment
 It allows you to follow the communication happening between the Koji platform and the template.
 
 [NOTE]
-The debugger is only available after you publish your template for the first time.
+The debugger is only useful for debugging published templates.
+If a template is unpublished, no data is available to the debugger.
 
-To open the debugger, go to *Developer Portal > My projects* and click the project. Then, click *Debugger* on the project overview page.
+From the Koji editor, you can open the debugger by clicking *Launch Debugger* in the preview pane.
```

Comment:
Emphasize how to open the debugger from the project details, rather than from the editor. e.g.,

"To open the debugger, go to *Developer Portal > My projects* and click the project. Then, click *Debugger* on the overview page. You can also open the debugger from the Koji editor by clicking …"